SessionState.js

```diff
@@ -0,0 +1,32 @@
+module.exports = class SessionState {
+
+  constructor(uuid, unit, completed=false, numPresses=1) {
```

Comment:
I'm thinking we should store the time we create a state - this will be important for replicating the twilio logic because some of the branches rely on a given amount of time spent without a response